spec/peep_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'peep'
+require 'database_helpers'
+
+describe Peep do
+  describe '.create' do
+    it 'creates a new peep' do
+      peep = Peep.create(message: 'This is a peep!')
+      persisted = persisted_data(id: peep.id)
+      
+      expect(peep).to be_a Peep
+      expect(peep.id).to eq persisted['id']
```

Comment:
I think I'd check that the persisted data message match, rather than checking the id match, given that the id is what you set on line 8.